### PR TITLE
Quick start improvements

### PR DESCRIPTION
**Pull request by davej-improbable**
- Made some steps and phrases more explicit so they're easier to follow.
- Added a table of K8s config variable names and example values to aid substitution.
- Replaced square brackets in config files with curly brackets and normalised the variable names to make automated substitution less error-prone.

### CODE REVIEW

File: docs/content/get-started/quickstart.md [hunk 1]
```diff
@@ -36,9 +35,9 @@ If you are using Google Cloud Platform, make sure you log in to it.
     - [Docker](https://docs.docker.com/install/) - to build the images.
     - [Google Cloud SDK](https://cloud.google.com/sdk/) - we use this tool to push built images up to our Google Cloud project.
     - [Terraform](https://www.terraform.io/) - we use this to configure the different cloud services we use.
-    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) - used to deploy services to the cloud Kubernetes instance. It's included in Docker Desktop if you're on Windows.
+    - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) - used to deploy services to the cloud Kubernetes instance. It's included in Docker Desktop if you're on Windows or Mac.
```
Comment (JoelOtter) [inline docs/content/get-started/quickstart.md]: Mac ->`macOS`

File: docs/content/get-started/quickstart-4.md [hunk 2]
```diff
@@ -52,15 +52,31 @@ kubectl create secret generic "spatialos-refresh-token" --from-file=./service-ac
...
+| `{{your_spatialos_project_name}}` | The name of your Spatial OS project | `alpha_hydrogen_tape_345` |
+| `{{your_playfab_title_id}}` | Your numeric Playfab Title ID | `123789` |
+| `{{your_redis_host}}` | The IP address of your Memory Store | `10.1.2.3` |
+| `{{your_gateway_host}}` | The IP address of your Gateway service | `123.4.5.6` |
+| `{{your_party_host}}` | The IP address of your Party service | `123.7.8.9` |
+| `{{your_playfab_auth_host}}` | The IP address of your Playfab Auth service | `123.10.11.12` |
+
+You can use `git grep "{{.*}}` to help find which files need editing.
+
+> In the real world you'll probably use a templating system such as Jinja2, or simply find-and-replace with `sed`, to do this step more easily. Kubernetes doesn't provide any templating tools out of the box so we haven't used any here; feel free to pick your favourite if you so choose. (For example, on UNIX systems you could use `git grep -lz "{{your_google_project_name}}" | xargs -0 sed -i '' -e "s/{{your_google_project_name}}/rhyming-pony-24680/g"`.)
```
Comment (JoelOtter) [inline docs/content/get-started/quickstart-4.md]: I think this is overly specific

File: docs/content/get-started/quickstart-4.md [hunk 3]
```diff
@@ -52,15 +52,31 @@ kubectl create secret generic "spatialos-refresh-token" --from-file=./service-ac
...
+| ---- | ----------- | ------------- |
+| `{{your_google_project_name}}` | The name of your Google Cloud project | `rhyming-pony-24680` |
+| `{{your_spatialos_project_name}}` | The name of your Spatial OS project | `alpha_hydrogen_tape_345` |
+| `{{your_playfab_title_id}}` | Your numeric Playfab Title ID | `123789` |
+| `{{your_redis_host}}` | The IP address of your Memory Store | `10.1.2.3` |
+| `{{your_gateway_host}}` | The IP address of your Gateway service | `123.4.5.6` |
+| `{{your_party_host}}` | The IP address of your Party service | `123.7.8.9` |
+| `{{your_playfab_auth_host}}` | The IP address of your Playfab Auth service | `123.10.11.12` |
+
+You can use `git grep "{{.*}}` to help find which files need editing.
```
Comment (JoelOtter) [inline docs/content/get-started/quickstart-4.md]: I didn't even know git came with grep...

File: docs/content/get-started/quickstart.md (hunk 1 above)
Comment (davej-improbable) [reply docs/content/get-started/quickstart.md]: We use "Mac" further further up the same document when referring to the SpatialOS tools installation link - should we use "macOS" instead there, too?

File: docs/content/get-started/quickstart-4.md (hunk 2 above)
Comment (davej-improbable) [reply docs/content/get-started/quickstart-4.md]: Fair enough, I was on the fence about including such an unwieldy line of code that most people would ignore anyway :)

File: docs/content/get-started/quickstart-4.md (hunk 3 above)
Comment (davej-improbable) [reply docs/content/get-started/quickstart-4.md]: Me neither until I was trying to find a safe way to `sed` inside a git repo without messing up files in `.git`! It ensures the only results returned are files in the repo, and adds highlighting too...

File: docs/content/get-started/quickstart-4.md (hunk 2 above)
Comment (davej-improbable) [reply docs/content/get-started/quickstart-4.md]: Changed!

File: docs/content/get-started/quickstart.md (hunk 1 above)
Comment (JoelOtter) [reply docs/content/get-started/quickstart.md]: Hmm. I think the main SpatialOS docs refer to macOS, so we should probably align with that. Would you mind making those changes too?

File: docs/content/get-started/quickstart-4.md (hunk 3 above)
Comment (JoelOtter) [reply docs/content/get-started/quickstart-4.md]: It's also (presumably) cross-platform. We can't guarantee the presence of grep on Windows but this should be there. Worth remembering!

File: docs/content/get-started/quickstart.md (hunk 1 above)
Comment (davej-improbable) [reply docs/content/get-started/quickstart.md]: No problem - done.

File: docs/content/get-started/quickstart-3.md [hunk 4]
```diff
@@ -12,12 +12,16 @@ You also need to ensure your `gcloud` client is authenticated properly:
...
 
-Now we can build and push our images. Navigate to the directory where the Dockerfiles are kept (`/services/docker`). We're going to use the `gateway` container image as an example, but you'll want to do this for each of the images `gateway`, `gateway-internal`, `party`, `playfab-auth` and `sample-matcher`.
+Now we can build and push the Docker images for our services. Navigate to the directory where the Dockerfiles are kept (`/services/docker`). We're going to build the images for each of the services we want to deploy, `gateway`, `gateway-internal`, `party`, `playfab-auth` and `sample-matcher`.
 
-Build the image like this:
+Build the images like this, replacing the `{{your_google_project_name}}` part with the name of your Google Cloud project:
 
 ```bash
-docker build -f ./gateway/Dockerfile -t "gcr.io/[your project id]/gateway" --build-arg CONFIG=Debug ..
+docker build -f ./gateway/Dockerfile -t "gcr.io/{{your_google_project_name}}/gateway" --build-arg CONFIG=Debug ..
```
Comment (JoelOtter) [inline docs/content/get-started/quickstart-3.md]: can we put line breaks between these?

Comment (davej-improbable) [reply docs/content/get-started/quickstart-3.md]: Done

**Pull request merged**